event_manager/models.py

```diff
@@ -1,3 +1,51 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.contrib import admin
+from django.contrib.auth.models import User
+from django.contrib.auth.admin import UserAdmin as AuthUserAdmin
+from django.http import HttpRequest
+
+class Member(models.Model):
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    netid = models.IntegerField(default=0)
+    role = models.CharField(max_length=200)
+    major = models.CharField(max_length=200)
+    points = models.IntegerField(default=0)
+
+    def __str__(self):
+        return self.user.get_full_name()
+
+class Team(models.Model):
+    title = models.CharField(max_length=200)
+    lead = models.ForeignKey(Member, null=True, on_delete=models.SET_NULL)
```

Comment:
We need to change this from Member to User, then we will need to make migrations again. For me, migrations were not changing anything, so I went into database manually and change member_id to user_id in the table event_manager_team_members